decidim-assemblies/app/controllers/decidim/assemblies/assemblies_controller.rb

```diff
@@ -5,7 +5,10 @@ module Assemblies
     # A controller that holds the logic to show Assemblies in a public layout.
     class AssembliesController < Decidim::Assemblies::ApplicationController
       include ParticipatorySpaceContext
-      participatory_space_layout only: :show
+
+      redesign active: false
```

Comment:
@entantoencuanto could this be removed?

Reply:
Yes, this was easy, this part is not necessary for the moment